### ISSUE TITLE
Try/except with pass showing "unreachable code" afterward

### ISSUE THREAD

**Issue by deregtd** [microsoft/pyright#163]
**Describe the bug**
If you have a try/except statement where the "try" clause has a return statement and the "except" clause has "pass", then all code after the block shows as unreachable in pyright

**To Reproduce**
```
def func():
    try:
        return int(5)
    except KeyError:
        pass

    return 5
```

**Expected behavior**
The "return 5" above should not show as "unreachable code".

**VS Code extension or command-line**
VSCode Extension 1.0.43

**Comment by erictraut** [comment]
This will be fixed in the next version of pyright.

**Comment by erictraut** [comment]
This is now fixed in 1.0.44, which I just published.
